src/pyodide/internal/setupPackages.ts

```diff
@@ -122,22 +126,18 @@ class SitePackagesDir {
  * This also returns the list of soFiles in the resulting site-packages
  * directory so we can preload them.
  */
-export function buildSitePackages(
-  requirements: Set<string>
-): [SitePackagesDir, boolean] {
+export function buildSitePackages(requirements: Set<string>): SitePackagesDir {
   const [bigTarInfo, bigTarSoFiles] = parseTarInfo();
 
-  let LOAD_WHEELS_FROM_R2 = true;
   let requirementsInBigBundle = new Set([...STDLIB_PACKAGES]);
-  if (bigTarInfo.children!.size > 10) {
-    LOAD_WHEELS_FROM_R2 = false;
+  if (!LOAD_WHEELS_FROM_R2 && !LOAD_WHEELS_FROM_ARTIFACT_BUNDLER) {
```

Comment:
Can you add a comment explaining this condition? It's not immediately clear what you are up to here. If we are using the packages built into the binary, then `requirementsInBigBundle` is the set of stdlib packages? Otherwise, it includes all requirements? So I guess the point is that the big bundle either has:
1. everything (currently) or
2. just standard library packages (when your autogate is on)?

If this isn't already explained in a comment somewhere, please explain it.

Reply:
This is meant as a transitionary thing, to support the current model as we're rolling it out. Eventually we are going to make the big bundle completely empty.

Reply:
Well it does kind of make sense to revendor the unvendored stdlib modules.